### ISSUE TITLE
Error: spawn C:\Users\zribi\OneDrive\Bureau\vroom-frontend\node_modules\.bin\browserify ENOENT

### ISSUE THREAD

**Issue by souhirzribi96** [VROOM-Project/vroom-frontend#58]
**when i run my front end app i found this error can you help me to found the solution of this**
`> vroom-frontend@0.3.0 serve C:\Users\zribi\OneDrive\Bureau\vroom-frontend
> beefy src/map.js --browserify ./node_modules/.bin/browserify --live

listening on http://localhost:9966/
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Users\zribi\OneDrive\Bureau\vroom-frontend\node_modules\.bin\browserify ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\Users\\zribi\\OneDrive\\Bureau\\vroom-frontend\\node_modules\\.bin\\browserify',
  path: 'C:\\Users\\zribi\\OneDrive\\Bureau\\vroom-frontend\\node_modules\\.bin\\browserify',
  spawnargs: [
    'C:\\Users\\zribi\\OneDrive\\Bureau\\vroom-frontend\\src\\map.js',
    '-d'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vroom-frontend@0.3.0 serve: `beefy src/map.js --browserify ./node_modules/.bin/browserify --live`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vroom-frontend@0.3.0 serve script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\zribi\AppData\Roaming\npm-cache\_logs\2021-03-24T14_27_19_238Z-debug.log
`

**Comment by jcoupey** [comment]
The `serve` script provided in package.json uses relative links to a local `browserify`:

https://github.com/VROOM-Project/vroom-frontend/blob/2f5484661685dd383e6c5a914f8527b14a8eb774/package.json#L7

It looks like the path don't really get correctly through on windows:
```
Error: spawn C:\Users\zribi\OneDrive\Bureau\vroom-frontend\node_modules.bin\browserify ENOENT
```

the `node_modules.bin` part is especially suspicious.

Can you try adjusting paths in package.json to see if it's any better?

**Comment by DCSnip3r** [comment]
I was able to reproduce this issue on Windows, and I made sure the filepath was correct
ie. `node_modules/.bin`

This may be a windows issue, or perhaps an issue with a dependency running on windows. This will work on Ubuntu, and my workaround was to run this in WSL2.

I am not sure what the solution will be. Does browserify need different file pointers for Windows environments? I'm unfamiliar with browserify but was seeing the > symbol online to map files for windows.
